### PR TITLE
Migrate Perfherder tests to Travis suite

### PR DESCRIPTION
**Pull request by davehunt**
Here I've renamed `test_treeherder_dropdown` to `test_switch_app` and cleaned it up a little. I've also moved the very basic `test_perfherder_main`from `test_basics.py` to `test_perfherder.py` and added a comment to indicate that more coverage should be added.

/cc @rbillings

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: tests/selenium/pages/treeherder.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
-from pypom import Page
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as expected
 
+from pages.base import Base
```
Comment (edmorley) [inline tests/selenium/pages/treeherder.py]: This looks like it's a relative import. I think it should be `from .base import Base` instead?

(Python 2 is quite lax about allowing not-technically-correct relative imports - as part of our prep for Python 3 ideally we'll sprinkle some `from __future__ import absolute_import` around, or at least start linting on Python 3)

File: tests/jenkins/pages/run.sh [hunk 2]
```diff
@@ -1,5 +0,0 @@
-#!/bin/bash -ex
-
-# shellcheck source=/dev/null
-source ./venv_modules/bin/activate
-firefox-ui-functional tests/test_tp.py --binary firefox/FirefoxNightly.app/Contents/MacOS/firefox-bin -v
```
Comment (edmorley) [inline tests/jenkins/pages/run.sh]: I'm guessing this is this now not used due to the switch to the new Jenkins environment?

Comment (davehunt) [reply tests/jenkins/pages/run.sh]: We're actually not sure what this is from, but it's never been used in relation to these tests.

File: tests/selenium/pages/treeherder.py (hunk 1 above)
Comment (davehunt) [reply tests/selenium/pages/treeherder.py]: I added `from __future__ import absolute_import` and kept the imports the same, and everything still passed. I then changed them to be relative such as `from .base import Base` without the `__future__` import and they still passed. I've pushed that latest change, but let me know if you would prefer something else.

**Pull request merged**